File: global_/environments/dockerfiles/Dockerfile.go
FROM golang:1.21-alpine

RUN apk add --no-cache \
    git \
    curl \
    build-base

RUN go install github.com/cosmtrek/air@latest 2>/dev/null || true && \
    go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest 2>/dev/null || true

WORKDIR /workspace

ENV GOPATH=/workspace/go

CMD ["/bin/sh"]

File: global_/environments/dockerfiles/Dockerfile.c
FROM gcc:latest

RUN apt-get update && apt-get install -y \
    gdb \
    make \
    cmake \
    valgrind \
    git \
    curl \
    build-essential \
    clang \
    clang-format \
    && rm -rf /var/lib/apt/lists/*

WORKDIR /workspace

CMD ["/bin/bash"]
